src/supported-locales.js

```diff
@@ -79,6 +79,15 @@ const customLocales = {
     }
 };
 
+const localeMap = {
```

Comment:
This feels a little cryptic. What's the difference between the keys and values here? They're both locale strings, but is this something like `mapScratchLocalesToTxLocales` or something?